.travis.yml

```diff
@@ -4,7 +4,10 @@ dist: trusty
 sudo: required
 
 go:
- - 1.13.8
+  - 1.13.8
+
+arch:
+  - amd64
```

Comment:
where is arm64?

Reply:
It doesn't pass yet (see my comment above). If we want to merge this before it passes, we can add it to `allow_failures`.

Reply:
> It doesn't pass yet (see my comment above). If we want to merge this before it passes, we can add it to `allow_failures`.
May I need to add allow_failures element here?

Reply:
> May I need to add allow_failures element here?

Yes, please go ahead :-)

Reply:
Done